### PR TITLE
remove fetch to login servlet from getAvailableEvents() to reduce number of fetch calls

### PR DESCRIPTION
**Pull request by Omarzintan**

### CODE REVIEW

**Review by Omarzintan — COMMENTED** [review]
Okay, this change does not reflect the fix. The fix is actually in the previous commit which involves removing a line from Events Servlet

File: src/main/webapp/mapScript.js [hunk 1]
```diff
@@ -216,21 +218,15 @@ function getPublicEvents() {
...
-
-  fetch("/login")
-    .then(response => response.json())
-    .then(json => {
-      userID = json['id'];
-    });
   fetch("events")
     .then(response => response.json())
     .then(events => {
+      console.log(events);
```
Comment (lexidelorey) [inline src/main/webapp/mapScript.js]: these are for testing I think ??

**Review by lexidelorey — DISMISSED** [review]

**Review by usaeedw — DISMISSED** [review]

File: src/main/webapp/mapScript.js (hunk 1 above)
Comment (Omarzintan) [reply src/main/webapp/mapScript.js]: Yhup, I will take that line out

**Comment by Omarzintan** [comment]
The most recent commits are to update map snippet function to derive place id and location name from create events page.

**Review by usaeedw — APPROVED** [review]

**Pull request merged**